src/testHarnesses/AdvancedLiquidityManagementTests.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.0;
+
+import { IRateLimits }           from "spark-alm-controller/src/interfaces/IRateLimits.sol";
+import { ChainIdUtils, ChainId } from "src/libraries/ChainId.sol";
+import { SpellRunner }           from './SpellRunner.sol';
+
+// TODO: expand on this on https://github.com/marsfoundation/spark-spells/issues/65
+abstract contract AdvancedLiquidityManagementTests is SpellRunner {
```

Comment:
ALM stands for Asset Liability Management btw. Also, we should call these `SparkLiquidityLayerTests`.